apps/balancer-tools/src/app/apr/layout.tsx

```diff
@@ -1,24 +1,28 @@
-"use client";
-
-import { useParams } from "next/navigation";
+import { Metadata } from "next";
 import * as React from "react";
-import invariant from "tiny-invariant";
 
 import { Header } from "#/components/Header";
 
 import HeaderEndButton from "./(components)/HeaderEndButton";
 
-export default function Layout({ children }: React.PropsWithChildren) {
-  const { roundId, poolId, network } = useParams();
-  invariant(!Array.isArray(roundId), "roundId cannot be a list");
-  invariant(!Array.isArray(poolId), "poolId cannot be a list");
-  invariant(!Array.isArray(network), "network cannot be a list");
+export const metadata: Metadata = {
+  title: "Balancer - APR Enhancement Calculator",
```

Comment:
@ribeirojose Could you validate this name?

Reply:
We can call it Historical APR